client/my-sites/media-library/footer.jsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import ReactDom from 'react-dom';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+import { takeRight, throttle } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ButtonGroup from 'components/button-group';
+import CompactCard from 'components/card/compact';
+import Count from 'components/count';
+import MediaActions from 'lib/media/actions';
+import MediaUtils from 'lib/media/utils';
+
+class MediaLibraryFooter extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			width: 0,
+		};
+	}
+
+	componentDidMount() {
+		this.throttleOnResize = throttle( this.calculateWidth, 200 );
+		window.addEventListener( 'resize', this.throttleOnResize );
+		this.calculateWidth();
+	}
+
+	componentWillUnmount() {
+		window.removeEventListener( 'resize', this.throttleOnResize );
+	}
+
+	calculateWidth = () => {
+		const { parent } = this.props;
+		const width = ReactDom.findDOMNode( parent ).clientWidth;
+		this.setState( { width } );
+	};
+
+	renderSelectedItem = ( item, index ) => {
+		const mimePrefix = MediaUtils.getMimePrefix( item );
+
+		const url = MediaUtils.url( item, {
+			photon: true,
+			maxWidth: 96,
+			size: 'small',
+		} );
+
+		if ( 'image' === mimePrefix ) {
+			return (
+				<div key={ index } className="media-library__footer-thumb">
+					<img src={ url } />
+				</div>
+			);
+		}
+
+		if ( 'video' === mimePrefix ) {
+			let videoThumb = null;
+			if ( this.props.media.thumbnails ) {
+				videoThumb = this.props.media.thumbnails.fmt_hd ||
+					this.props.media.thumbnails.fmt_dvd ||
+					this.props.media.thumbnails.fmt_std;
+			}
+
+			if ( videoThumb ) {
+				return (
+					<div key={ index } className="media-library__footer-thumb">
+						<img src={ url } />
+					</div>
+				);
+			}
+
+			return (
+				<div key={ index } className="media-library__footer-thumb">
+					<Gridicon icon="video" size={ 48 } />
+				</div>
+			);
+		}
+
+		return (
+			<div key={ index } className="media-library__footer-thumb">
+				<Gridicon icon="pages" size={ 48 } />
+			</div>
+		);
+	};
+
+	onCancel = () => {
+		const { site } = this.props;
+		MediaActions.setLibrarySelectedItems( site.ID, [] );
+	};
+
+	render() {
+		const { selectedItems, onViewDetails, onDeleteItem, translate } = this.props;
+
+		if ( ! selectedItems || ! selectedItems.length ) {
+			return null;
+		}
+
+		return (
+			<CompactCard className="media-library__footer" style={ { width: this.state.width + 'px' } }>
+				<div className="media-library__footer-buttons">
+					<Button onClick={ this.onCancel }>{ translate( 'Cancel' ) }</Button>
+				</div>
+				<div className="media-library__footer-count">
+					<Count count={ selectedItems.length } />
+					<span className="media-library__footer-count-label">{ translate( 'Selected' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **4** times:
`translate( 'selected' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/magnificient-media-header)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).